### PR TITLE
Refine the top level README

### PR DESCRIPTION
**Pull request by sdake**
- clean up the language to use active voice
- remove conjunctions which may cause confusion
- add awesome emojis

Signed-off-by: Steven Dake <steven.dake@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
+The [TODO Group](https://todogroup.org) is an open community of practitioners who aim to create and share knowledge, collaborate on practices, tools, and other ways to run successful and effective Open Source Program Offices or similar Open Source initiatives. TODO Group is formed by its [Community participants](https://todogroup.org/community) and [General Members](https://todogroup.org/members).
 
-Everyone is welcome to attend the monthly TODO Group OSPOlogy meeting: https://community.linuxfoundation.org/todo-group/
+## 🦺 Community
 
-## TODO EU Chapter Sync Meetings (monthly)
+To paritipate in OSPO related discussions please [submit questions](https://github.com/todogruop/ospology/discussions) or join the [TODO Group Slack](https://slack.todogroup.org).
 
-The TODO EU chapter was formed by TODO Members based in Europe. This chapter was created with the aim of working together to improve OSPO adoption and education within Europe and discuss with the broader community the challenges European organizations face when implementing an open source program.
+### 🌎 North America Community Meeting
```
Comment (anajsana) [inline README.md:14]: TODO Community meetings structure changes a bit from what is has been described here, let me explain this better 🙂 They are structured into:

* [OSPOlogy community meetings](https://github.com/todogroup/ospology/tree/main/meetings#ospology-monthly-meetings): These are "General and public monthly meetings of the TODO Group and wider OSPO community, everyone is welcome to attend and participate". These meetings have always been a featured topic with special guests and the last 15 mins are reserved for community discussions. all these meetings are recorded, edited, and recorder on Youtube OSPOlogy Channel, so everyone across the globe can have access to it 👍 https://github.com/todogroup/ospology/tree/main/meetings#event-page-and-past-sessions

* TODO Chapters: These are location-based meetings and they do not follow the same structure as OSPOlogy community meetings. The community put topics of matter into an agenda and we have discussions the entire hour. We currently have [Europe Sync meetings](https://github.com/todogroup/ospology#todo-eu-chapter-sync-meetings-monthly) and [APAC Sync meetings](https://github.com/todogroup/ospology/discussions/67). Both are available in the community page calendar: https://todogroup.org/community/

**Review by anajsana — CHANGES_REQUESTED** [review]
This is great!! Thank you so much for contributing to making the README.md more appealing 💯 I've requested some changes in some of the sections and added comments between lines 🙂

**Comment by anajsana** [comment]
Just reviewing opened PRs at ospology and checking their status. @sdake do you need help with this or is there anything I can help with? 🙂

**Comment by sdake** [comment]
Ana,

I will finish this PR today. Thanks for the patience.

**Comment by anajsana** [comment]
> Ana, I will finish this PR today. Thanks for the patience.

@sdake are there any updates on this? 🙂

**Comment by sdake** [comment]
Thanks, I have been a little busy with IRL. I will try to finish this easy
task up this week.

Cheers,
-steve

**Review by anajsana — APPROVED** [review]

**Comment by anajsana** [comment]
@sdake I committed to your branch the requested changes to the readme.md in order to move forward with this. However, there were a lot of previous changes made to the readme since you created this PR, and seems that this branch cannot be rebased due to conflicts.

**Comment by anajsana** [comment]
Since this PR has not shown any activity over the past months, I'll create a new PR with the changes and mention this PR there.

**Comment by anajsana** [comment]
Since this PR has not shown any activity over the past months, I'll create a new PR with the changes and mention this PR there https://github.com/todogroup/ospology/pull/136

**Comment by sdake** [comment]
Thanks @anajsana. I am attempting to find a new job, and I have been busy with a job hunt. Thank you for your understanding. I hope some of the ideas were useful. Also, nice job on adding netlify to the build system!

Cheers,
-steve

**Pull request closed without merging**